language/v2/language_entities_gcs.py

```diff
@@ -0,0 +1,115 @@
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "language_entities_gcs")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-language
+
+# sample-metadata
+#   title: Analyzing Entities (GCS)
+#   description: Analyzing Entities in text file stored in Cloud Storage
+#   usage: python3 samples/v2/language_entities_gcs.py [--gcs_content_uri "gs://cloud-samples-data/language/entity.txt"]
+
+# [START language_entities_gcs]
+from google.cloud import language_v2
+
+
+def sample_analyze_entities(gcs_content_uri):
```

Comment:
can you add type hints for the variable and the return type?

Reply:
This PR should follow the v1 sample here - https://cloud.google.com/natural-language/docs/analyzing-entities#analyzing_entities_in_a_string

Reply:
new samples should adhere to newer style guides, and consistency won't be the best argument for going back on older styles 😓

Reply:
Agreed. Added the hint types.